src/Globe/TileMesh.js

```diff
@@ -128,32 +168,58 @@ define('Globe/TileMesh', [
         }
     };
 
+    TileMesh.prototype.setDisplayed = function(show) {
+
+        //this.material.visible = show;
```

Comment:
Remove dead code :)
(and there are a bunch of blank lines to remove as well in this file)